ddlib/ddlib/util.py

```diff
@@ -98,7 +98,7 @@ def normalize_type_name(ty):
   if ty in CANONICAL_TYPE_BY_NAME:
     return CANONICAL_TYPE_BY_NAME[ty]
   else:
-    for patt,ty_canonical in CANONICAL_TYPE_BY_REGEX.iteritems():
+    for patt,ty_canonical in CANONICAL_TYPE_BY_REGEX.items():
```

Comment:
Seems like we're sacrificing Python2 performance for code compatibility with Python3.. Is there a way to still use generators in py2 for simple iterations?  Or is there proof that the difference is negligible?